lib/person.rb

```diff
@@ -0,0 +1,43 @@
+require 'yaml'
+require 'date'
+require 'pry'
+require './lib/library.rb'
+
+class Person
+
+attr_accessor :books
+
+def initialize
+  @books = YAML.load_file('./lib/person.yml')
+end
+
+def perform_returnbook(title)
+  @books.detect do |x|
+    if x[:item][:title] == title
+      index = @books.index(x)
+      @books.delete_at(index)
+      # File.open('./lib/person.yml', 'w') { |f| f.write @books.to_yaml }
+    end
+  end
+end
+
+def watch_list(library)
+  library.see_list
+end
```

Comment:
I'm surprised that `puts` method works the way you think it will. Instead of using `puts x`, you can just say `x` - the implicit return is what is allowing you to see the books... if you are. Is this method tested and are you sure it's doing what you think it's doing?